clang/include/clang/Sema/Sema.h

```diff
@@ -12240,16 +12240,16 @@ class Sema final : public SemaBase {
     bool PrevLastDiagnosticIgnored;
 
   public:
-    explicit SFINAETrap(Sema &SemaRef, bool AccessCheckingSFINAE = false)
+    explicit SFINAETrap(Sema &SemaRef, bool TestWellformedSFINAE = false)
```

Comment:
Can you add a comment for the parameter?

Reply:
Maybe `ForValidityCheck` would be clearer.

Reply:
Adding a comment makes sense to me. I'm open to a different name, but I started with "well-formed" because that's how the standard calls it. "Valid" sounds fine, but it doesn't seem to be what the standard calls it.

Reply:
Although later it does say "only the validity of the immediate context is considered." So I guess it's fine.